docs/content/how-to/mce/Dual/hypervisor/network-manager-dispatcher.md

```diff
@@ -0,0 +1,61 @@
+This is a script which modifies the system DNS resolver to point firstly to the dnsmasq (configured later). This will allow the virtual machines to resolve the different domains, routes and registries you will need for the different steps of the process.
+
+To enable this, we need to create an script in `/etc/NetworkManager/dispatcher.d/` called `forcedns`, and it will have this content:
```

Comment:
```suggestion
To enable this, we need to create a script in `/etc/NetworkManager/dispatcher.d/` called `forcedns`, and it will have this content:
```